src/hotspot/share/ci/ciField.cpp

```diff
@@ -253,6 +253,15 @@ static bool trust_final_non_static_fields(ciInstanceKlass* holder) {
   return TrustFinalNonStaticFields;
 }
 
+const char* stable_value_klass_name = "java/lang/StableValue";
+
+static bool trust_final_non_static_fields_of_type(Symbol* signature) {
+  if (signature->equals(stable_value_klass_name) == 0) {
```

Comment:
Just noticed that this usage can be updated to using
```
if (signature == vmSymbols::java_lang_StableValue)
```
once we add the constant.

Reply:
Sorry for my earlier misleading comments, but I just realized that we need a separate `vmSymbols` declartion besides `vmSymbols::java_lang_StableValue`: a `vmSymbols::java_lang_StableValue_signature`, with value `Ljava/lang/StableValue;` for field signatures (descriptors).

Since we are in `ciField.cpp`, we should use `ciSymbols::java_lang_StableValue_signature` (as this is a signature, not owner class) instead. (ciSymbols imports everything from vmSymbols with C++ templates)

Reply:
I've managed  to fix it using `vmSymbols::java_lang_StableValue_signature()` but ciSymbols methods are of another type.